_components/site-alert.md

```diff
@@ -0,0 +1,97 @@
+---
+permalink: /components/site-alert/
+redirect_from:
+- /site-alert/
+- /components/site-alerts/
+layout: styleguide
+type: component
+title: Site Alert
+category: Components
+lead: A site alert communicates urgent sitewide information.
+---
+
+When your agency has urgent information to share with the public, it’s essential that it stands out from other content on the site. A site alert prominently displays critical, time-sensitive warnings or directions across every page so that users see it whenever they visit the site. Using the same site alert treatment across agencies helps create a consistent and predictable way for users to find urgent information across all government websites. 
+
+{% include code/preview.html component="site-alert" %}
+
+<section class="site-component-section">
+  {% include code/accordion.html component="site-alert" %}
+  <div class="usa-accordion usa-accordion--bordered site-accordion-docs">
+    <button class="usa-button-unstyled usa-accordion__button"
+        aria-expanded="true" aria-controls="alert-docs">
+      Guidance
+    </button>
+    <div id="alert-docs" aria-hidden="false" class="usa-accordion__content site-component-usage">
+      <h4>When to use the site alert component</h4>
+      <ul class="usa-content-list">
+        <li><strong>Critical system notifications.</strong> Use a site alert to deliver a static system status update, such as notices about unavailable services or content. Site alerts should appear by default and not in response to an action.</li>
+        <li><strong>Messages that should be displayed on every page.</strong> A site alert makes critical information obvious and findable on every page.</li>
+      </ul>
+      <h4>When to consider something else</h4>
+      <ul class="usa-content-list">
+        <li><strong>Responding to user actions.</strong> Use the <a href="https://designsystem.digital.gov/components/alert/">alert</a> component for page-level validation messages (like error messages on a form) or to inform a user that a task was completed successfully.</li>
```

Comment:
Did you mean to make this an absolute link? I thought the URLs for links within the design system are something like `href="{{ site.base_url }}/components/alert"`.

Do we add titles to your `a` tags?

Reply:
updated this in both my PRs. Great catch!